src/main/java/it/gov/pagopa/apiconfig/selfcareintegration/model/channel/ChannelDetails.java

```diff
@@ -0,0 +1,252 @@
+  private String brokerDescription;
+
+  @JsonProperty("password")
+  private String password;
+
+  @JsonProperty("new_password")
+  private String newPassword;
+
+  @JsonProperty("protocol")
+  @NotNull
+  private Protocol protocol;
+
+  @JsonProperty("ip")
+  private String ip;
+
+  @Min(1)
+  @Max(65535)
+  @JsonProperty("port")
+  @NotNull
+  private Long port;
+
+  @JsonProperty("service")
+  private String service;
+
+  @JsonProperty("broker_psp_code")
+  @NotBlank
+  private String brokerPspCode;
+
+  @JsonProperty("proxy_enabled")
+  private Boolean proxyEnabled;
+
+  @JsonProperty("proxy_host")
+  private String proxyHost;
+
+  @Min(1)
+  @Max(65535)
+  @JsonProperty("proxy_port")
+  private Long proxyPort;
+
+  @JsonProperty("proxy_username")
+  private String proxyUsername;
+
+  @ToString.Exclude
+  @JsonProperty("proxy_password")
+  private String proxyPassword;
+
+  @JsonProperty("target_host")
+  private String targetHost;
+
+  @JsonProperty("target_port")
+  private Long targetPort;
+
+  @JsonProperty("target_path")
+  private String targetPath;
+
+  @Min(1)
+  @JsonProperty("thread_number")
+  @NotNull
+  private Long threadNumber;
+
+  @Min(0)
+  @JsonProperty("timeout_a")
+  @NotNull
+  @Builder.Default
+  private Long timeoutA = 15L;
+
+  @Min(0)
+  @JsonProperty("timeout_b")
+  @NotNull
+  @Builder.Default
+  private Long timeoutB = 30L;
+
+  @Min(0)
+  @JsonProperty("timeout_c")
+  @NotNull
+  @Builder.Default
+  private Long timeoutC = 120L;
+
+  @JsonProperty("nmp_service")
+  private String nmpService;
+
+  @JsonProperty("new_fault_code")
+  private Boolean newFaultCode;
+
+  @JsonProperty("target_host_nmp")
+  private String targetHostNmp;
+
+  @JsonProperty("target_port_nmp")
+  private Long targetPortNmp;
+
+  @JsonProperty("target_path_nmp")
+  private String targetPathNmp;
+
+  @JsonProperty("redirect_ip")
+  private String redirectIp;
+
+  @JsonProperty("redirect_path")
+  private String redirectPath;
+
+  @Min(1)
+  @Max(65535)
+  @JsonProperty("redirect_port")
+  private Long redirectPort;
+
+  @JsonProperty("redirect_query_string")
+  private String redirectQueryString;
+
+  @JsonProperty("redirect_protocol")
+  private Protocol redirectProtocol;
+
+  @JsonProperty("payment_model")
+  @Schema(required = true)
+  @NotNull
+  private PaymentModel paymentModel;
+
+  @JsonProperty("serv_plugin")
+  private String servPlugin;
+
+  @JsonProperty("rt_push")
+  @Schema(required = true)
+  @NotNull
+  private Boolean rtPush;
+
+  @JsonProperty("on_us")
+  @Schema(required = true)
+  @NotNull
+  private Boolean onUs;
+
+  @JsonProperty("card_chart")
+  @Schema(required = true)
+  @NotNull
+  private Boolean cardChart;
+
+  @JsonProperty("recovery")
+  @Schema(required = true)
+  @NotNull
+  private Boolean recovery;
+
+  @JsonProperty("digital_stamp_brand")
+  @Schema(required = true)
+  @NotNull
+  private Boolean digitalStampBrand;
+
+  @JsonProperty("flag_io")
+  private Boolean flagIo;
```

Comment:
@aomegax  do we need to add `flag_psp_cp`?

Reply:
Updated and added on response model